src/infrastructure/getMosaic.js

```diff
@@ -46,19 +50,55 @@ class sdkMosaic {
     return mosaicAmount
   }
 
-  static getMosaicInfo = async mosaicHexOrNamespace => {
-    let mosaicID = ''
+// TODO(ahuszagh) Remove...
+//  static getMosaicInfo = async mosaicHexOrNamespace => {
```

Comment:
IMO, we may need to accept `mosaicHex` or `Namespace` as params.
example `308f144790cd7bc4` is mosaicHex.

but some mosaic are link with then `namespace` such as `nem.xem`

so the params should accept `nem.xem` and convert to `308f144790cd7bc4`, then only request `/mosaics/from/${mosaicId}/limit/${limit}`

Reply:
@AnthonyLaw We're going to be paginating this from a list of mosaic info models, not mosaic names. We have the ID readily available, getting the name requires a lot of spurious requests.